Header Header

int16 leftY
int16 rightY

int16 A
int16 B
int16 X
int16 Y 
int16 back
int16 start
int16 guide
int16 leftBumper
int16 rightBumper


float32 leftTrigger
float32 rightTrigger

float32 normalizeRightY
float32 normalizeLeftY
